modules/openshift-architecture-common-terms.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-architecture-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+Access policies::
+A set of roles that dictate how users, applications, and entities within a cluster interacts with one another. Access policies increase cluster security.
+
+Admission plug-ins::
```

Comment:
Not sure if we have terms defined elsewhere in the doc set, but just a general suggestion for all of these would be to lowercase terms (i.e. show the correct capitalization). 

Like:

```
access policies:: ...

admission plug-ins:: ...
```

Looking at these, users might think that it should be written as capitalized "Admission plug-ins" even in the middle of a sentence. 

The IBM style guide and SSG both use lowercase/proper capitalization when listing their words. And the IBM style guide gives this guidance: 

> Do not capitalize a term in the glossary unless it is a proper noun.

Reply:
acked

Reply:
@bergerhoffer Does the capitalization look correct now? I made some changes to the terms. Seems like we have a merged instance of glossary items here: https://docs.openshift.com/container-platform/4.11/installing/index.html#install-openshift-common-terms_ocp-installation-overview